trace/table_models/curve_model.py

```diff
@@ -187,6 +224,22 @@ def set_data(self, column_name: str, curve: BasePlotCurveItem, value: Any) -> bo
         logger.debug("Finished setting curve data")
         return ret_code
 
+    def flags(self, index):
```

Comment:
can you add the parameters and return to the docstring?

Reply:
Solved this issue 👍